### ISSUE TITLE
Background extra animated during removing cells in the full swiping mode

### ISSUE THREAD

**Issue by c-villain** [c-villain/SwipeActions#31]
![Simulator Screenshot - iPhone 15 Pro Max - 2024-09-05 at 17 29 15](https://github.com/user-attachments/assets/d5d7122e-f580-4ec8-b4c8-48ae46cd48f5)

**Comment by c-villain** [comment]
New engine was added to swipeactions.
To control view's id you should use modifier `.identifier(your id)`:

```swift
ForEach(...) { cell in
     YourView(cell)
         .addSwipeAction(...) {}
         .identifier(cell) // <= ✅ Look here
 }
```

**Comment by c-villain** [comment]
fix in [0.4.0](https://github.com/c-villain/SwipeActions/releases/tag/0.4.0)
